.travis.yml

```
@@ -1,13 +1,30 @@
 language: node_js
 node_js:
   - "10"
-
+services:
+  - docker
 cache:
+  yarn: true
   directories:
-    - "node_modules"
+    - node_modules
 
-install:
-  - yarn install
+before_script:
```

```suggestion
install:
```
@appinteractive I guess if you remove `install` entirely, Travis will run `yarn install` by default. What happens if we simply build the docker image during `install`?